stun-types/src/attribute/integrity.rs

```diff
@@ -148,7 +148,7 @@ impl MessageIntegrity {
         })?;
         hmac.update(data);
         hmac.verify_slice(expected).map_err(|_| {
-            error!("integrity check failed");
+            debug!("integrity check failed");
```

Comment:
CI errors are relevant. Need to actually import `tracing::debug`.

Reply:
my bad, haven't checked the build status